modules/exploits/windows/fileformat/videocharge_studio.rb

```diff
@@ -19,6 +19,12 @@ def initialize(info = {})
           machine by enticing a user of VideoCharge Studio to open a malicious .VSC file.
       },
       'License'    => MSF_LICENSE,
+      'Notes' => {
+        'Stability' => [],
+        'SideEffects' => [],
+        'Reliability' => [],
+        'NOCVE' => 'Reason not given'
```

Comment:
Is this meant to be an array or a string?

https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/linux/upnp/belkin_wemo_upnp_exec.rb#L63

https://github.com/rapid7/metasploit-framework/blob/05d9e7f9d6af88b430247dfc22ce63a9a9853f70/modules/exploits/unix/webapp/bolt_authenticated_rce.rb#L86

Should probably add a check for that